tests/src/ExampleAttributesCommandFile.php

```diff
@@ -0,0 +1,42 @@
+<?php
+namespace Consolidation\TestUtils;
+
+use Consolidation\AnnotatedCommand\CommandAttributes;
+
+/**
+ * Test file used in the Annotation Factory tests.  It is also
+ * discovered in the testCommandDiscovery() test.
+ *
+ * The testCommandDiscovery test search base is the 'src' directory;
+ * any command files located immediately inside the search base are
+ * eligible for discovery, and will be included in the search results.
+ */
+class ExampleAttributesCommandFile
+{
+    protected $state;
+    protected $output;
+
+    public function __construct($state = '')
+    {
+        $this->state = $state;
+    }
+
+    public function setOutput($output)
+    {
+        $this->output = $output;
+    }
+
+    #[CommandAttributes(
+        description: 'This is the my:echo command',
+        help: "This command will concatenate two parameters. If the --flip flag\nis provided, then the result is the concatenation of two and one.",
+        aliases: ['c'],
+        usage: ['bet alpha --flip' => 'Concatenate "alpha" and "bet".']
+    )]
```

Comment:
This CommandAttributes structure looks pretty good to me. The only question I have is what do we want to name it? `#[CLI(...)]`?  CommandAttributes is also fine.

Reply:
no strong feelings, so when this is ready to merge, let's bikeshed 🙃

Reply:
As I recall multi-line attributes are not BC for syntax on PHP <8.0

Reply:
About naming DrushCommands looks better name

Reply:
Regarding the color of the CommandAttributes shed, I think that the term "Command" is very general and could refer to a wide variety of different sorts of commands. I think I'd therefore prefer "CommandLine" or "CommandLineAttributes". Not sure I am still enamored of my initial suggestion "CLI".

Regarding multi-line attributes, I think that we should recommend them by default. I don't think anyone is going to want to use command line attributes at all unless they are being used in a context where php 8 is required. Otherwise, you'd need both the php 8 attribute syntax and the legacy syntax at the same time, and we'd have to make sure that annotated command would not get confused and register the same command twice in such a scenario. Better to use only the legacy syntax until php 8 is your minimum supported version.

Regarding the name "DrushCommands", whatever we choose here is how Robo commands and any CLI tool derived from Robo will be annotated, so we need a general name. As follow-on work we could consider allowing derived applications to extend our command line attribute class. I don't think it would be worth it to do that unless the app wanted to extend the definition, but in that instance I expect it would be preferable to just define a second annotation. I don't have a strong opinion on this, but it doesn't seem too important at the moment.

Reply:
I went with `CommandLineAttributes`. renaming is a small change, so further discussion is welcome.